RIOT-OS/RIOT
# doc: Add guide for dev setup on Windows

### Contribution description

This adds a detailed instruction on how to get a working development setup on Windows.

State

- [x] Installation of WSL/Ubuntu
- [x] Cloning RIOT + first steps
- [x] Installation of VS Code
- [x] Integration of VS Code to WSL + IntelliSense
- [x] Working and compiling from VS Code
- [x] Working on real hardware via USB IP

### Testing procedure

[Check here](https://github.com/maribu/RIOT/tree/doc/getting-started/windows/doc/guides/setup-windows) for a rendering of the added README.md

### Issues/PRs references

https://github.com/RIOT-OS/RIOT/issues/20388
